CHANGELOG.md

```diff
@@ -5,6 +5,13 @@
   - Shorten timeout for connection, add retries, and use persistent HTTP connections.
 - Drop support for Faraday < 1.9.
 
+## [7.2.0](https://github.com/mdsol/mauth-client-ruby/compare/v7.1.0...v7.2.0) (2024-04-25)
```

Comment:
Oops

Reply:
no worries, release-please is going to create the `v7.2.0` tag 😉

Reply:
My "Oops" was to say that release-please inserted 7.2.0 changes below 7.1.0 changes